distribution/common/src/main/resources/etc/org.codice.ddf.distribution.landing-page.properties.config

```diff
@@ -0,0 +1,2 @@
+description="Alliance extends DDF and adds NATO Compliant Applications"
```

Comment:
Use what's on the README for now:
"Codice Alliance is an open source, modular integration framework building on DDF with the addition of DoD/IC and NATO support capabilities."